gradlew

```diff
@@ -0,0 +1,234 @@
+#!/bin/sh
+
+#
+# Copyright © 2015-2021 the original authors.
```

Comment:
Is this copyright ok?

Reply:
I copied this exec from Etendo Core. Should I update the copyright info?